python/Ganga/new_tests/GPI/GangaList/TestCopy.py

```diff
@@ -43,7 +36,7 @@ def testCopy(self):
         gl2 = copy.copy(gl)
         assert len(gl2) == len(gl), 'lists must be equal'
         assert gl2 is not gl, 'list must be copies'
-        #assert gl[0] is gl2[0], 'the references must be copied' ## rcurrie 09/12/15 Not sure this is valid given we have to cope with GangaObjects here which may not be copyable!
+        assert gl[0] is gl2[0], 'the references must be copied'
```

Comment:
Disallowing a ```copy``` would cause more problems imo. I think the above behavior is sensible. It's impossible (I hope) to assign the reference to another which can be a parent without a deepcopy being made within Ganga atm and many tests would at least need restructuring with copy being disabled in front of the proxy.
I think if the user does this and is surprised it's a probably good learning opportunity for them to learn some Python features/behaviour.